src/CMakeLists.txt

```diff
@@ -32,25 +25,36 @@ set(NOVELRT_LIB_LINK_LIBRARIES
   spdlog::spdlog
 )
 
-if(NOT WIN32)
-  set(NOVELRT_LIB_LINK_LIBRARIES
-    ${NOVELRT_LIB_LINK_LIBRARIES}
-    m
-    stdc++fs
-  )
-endif()
+# lua doesn't provide native cmake support
+
+find_path(LUA_INCLUDE_DIRS lua.h)
+include_directories(${LUA_INCLUDE_DIRS})
+
+find_library(LUA_LIBRARY NAMES liblua lua)
+link_libraries(${LUA_LIBRARY})
+
+# nethost doesn't provide native cmake support
+
+find_path(NETHOST_INCLUDE_DIRS nethost.h)
+include_directories(${NETHOST_INCLUDE_DIRS})
+
+find_library(NETHOST_LIBRARY NAMES libnethost nethost)
+link_libraries(${NETHOST_LIBRARY})
 
 if (WIN32)
   set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Is there a reason this is WIN32 specific?

Reply:
Linux automatically does the right thing.

Reply:
Ah oki doki. Resolving.